translations/base-en.yaml

```diff
@@ -519,11 +519,11 @@ buildings:
     underground_belt:
         default:
             name: &underground_belt Tunnel
-            description: Allows you to tunnel resources under buildings and belts.
+            description: Allows you to tunnel resources up to <strong>5</strong> blocks away under buildings and belts. 
```

Comment:
Numbers (just numbers) are not bolded anywhere else in the translation file; I can see why bolding would make sense, but what about lines 714 and 795?
https://github.com/tobspr/shapez.io/blob/fa8857c6242a5a9958f59c4345c982f47613cb5b/translations/base-en.yaml#L714
https://github.com/tobspr/shapez.io/blob/fa8857c6242a5a9958f59c4345c982f47613cb5b/translations/base-en.yaml#L795